src/config/config.ts

```diff
@@ -116,7 +116,9 @@ import { isArray, isDefined, isFunction, isObject } from '../util/util';
  * | `pickerEnter`            | `string`            | The name of the transition to use while a picker is presented.                                                                                   |
  * | `pickerLeave`            | `string`            | The name of the transition to use while a picker is dismissed.                                                                                   |
  * | `popoverEnter`           | `string`            | The name of the transition to use while a popover is presented.                                                                                  |
- * | `popoverLeave`           | `string`            | The name of the transition to use while a popover is dismissed.                                                                                  |
+ * | `popoverLeave`           | `string`            | The name of the transition to use while a popover is dismissed. 
+ * | `scrollAssist`           | `boolean`           | Used to avoid the input to be hidden by the keyboard if it's near the bottom of the page. 
```

Comment:
Not sure these are props we want out in the public. 
Since these shouldn't be touched by devs.

Reply:
I found these properties on some answers on deferent  forums and it resolved some wired issue, So if possible make it public.